config/site.toml

```diff
@@ -1,8 +1,8 @@
 title = "Spin Docs"
-# base_url = "https://spinframework.com"
+# base_url = "https://spinframework.dev"
```

Comment:
This is commented out and can be removed?  I am not sure what it does or if the empty one on the next line is okay though

Reply:
Right, I'm not sure either... maybe @karthik2804 recalls?

Reply:
The base_url does not seemed to be used for anything other than an eprintln it looks like 
https://github.com/search?q=repo:fermyon/bartholomew%20base_url&type=code,

So I would assume that it is safe to remove.